src/main/java/com/cloudant/client/api/views/ViewResponse.java

```diff
@@ -140,6 +140,28 @@
     ViewResponse<K, V> previousPage() throws IOException;
 
     /**
+     * Returns an encoded string representing the query for the next page of results, which can
+     * be used with {@link ViewRequest#getResponse(String)} to retrieve the next page without
+     * keeping ViewResponse state.
+     *
+     * @return encoded string parameter for the next page, or null if there is no next page
```

Comment:
If you chose to use `pagination token` this can be made simpler, "opaque pagination token for next page, or null...".